cmd/oras/internal/option/remote_test.go

```diff
@@ -415,3 +416,56 @@ func TestRemote_parseCustomHeaders(t *testing.T) {
 		})
 	}
 }
+
+func TestRemote_NewRepository_Retry(t *testing.T) {
+	caPath := filepath.Join(t.TempDir(), "oras-test.pem")
+	if err := os.WriteFile(caPath, pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: ts.Certificate().Raw}), 0644); err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	retries, count := 3, 0
+	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		count++
+		if count < retries {
+			w.Header().Set("Retry-After", "1")
```

Comment:
Is `Retry-After` a specific header used by underlying retry package? Didn't see its usage here.

Reply:
It is not used, but it is a header usually set when you are rated limited. I will remove it as it is not needed here.